test/rubygems/test_gem_printable_uri.rb

```diff
@@ -0,0 +1,80 @@
+require_relative 'helper'
+require 'uri'
```

Comment:
Do we need to still require `uri` here?

Reply:
Good question. I have a couple of tests to make sure we can pass a URI object to this class and it will work. but now I think it's unnecessary since we will be passing a string to this method.

Reply:
Thanks, @simi. I've removed it.